RadixWallet/Features/ImportMnemonic/ImportMnemonic.swift

```diff
@@ -60,6 +60,9 @@ public struct ImportMnemonic: Sendable, FeatureReducer {
 			words.compactMap(\.completeWord)
 		}
 
+		/// irrelevant for readonly
+		public var isProgressing = false
```

Comment:
hard to name this one, cannot be named `isSavingMnemonic` since if `persistStrategy` is `nil`, we are in fact not saving. Hmm maybe could be named `isImportOfMnemonicInProgress` ? but I think this is OK a name.

Reply:
`var isSavingMnemonic: Bool? = nil`

Reply:
sure, changing in my other mnemonics PR https://github.com/radixdlt/babylon-wallet-ios/pull/878